### PR TITLE
[MNG-6113] Rename the 'Central Repository' to 'Maven Central Reposito…

### PR DESCRIPTION
**Pull request by michael-o**
…ry' in the 4.0.0 super POM

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MNG-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MNG-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

**Comment by rfscholte** [comment]
@hboutemy has a strong opinion about this.

**Comment by michael-o** [comment]
I'd love to here it. I would expect that we use a canonical name which carries your name: Maven.

**Comment by michael-o** [comment]
@hboutemy What is your opinion.

**Comment by hboutemy** [comment]
Central Repository is used not only by Maven
and "Maven Central" is a registered service name as stated in https://issues.apache.org/jira/browse/MNG-5164

**Comment by michael-o** [comment]
Central has been initiated by Maven devs and even non-Maven users refer to it Maven Central. Are you saying that Central Repository is a trademark? I had this discussions once with Mark Thomas, the ASF does not hold any registrations as far as I remember. I did not fully understand what you were trying to say.

**Comment by rfscholte** [comment]
I think I understand the issue here. Central contains artifacts: they don't have to be created by Maven, nor do they have to be consumed by Maven projects only. 
For similar reason I chose to create plexus-languages/plexus-java: it might look weird if it was called maven-java and used by projects that have nothing to do with Maven , e.g. other build tools.

**Comment by cstamas** [comment]
AFAIK, it *was* Maven Central before "re-branding", even the book mentions it as such, oh, and btw, it is in `maven.org` domain as well :smile:

**Comment by michael-o** [comment]
@michael-o The layout, structure and meta files are dictated by the Maven community,  maven-metadata.xml. If you claim this is a non-Maven repo I request to rename those files.

**Comment by mosabua** [comment]
I has been operated under the name "Central Repository" by Sonatype for well over a decade. I would leave the name alone

https://central.sonatype.org/

cc @brianf

**Comment by cstamas** [comment]
Well, https://search.maven.org/ names it differently 😄

**Comment by cstamas** [comment]
> Central Repository is used not only by Maven
> and "Maven Central" is a registered service name as stated in https://issues.apache.org/jira/browse/MNG-5164

@hboutemy seems it WAS registered, but is abandoned (since 2013?). This argument, actually both of them, are fluke IMHO.

**Review by eolivelli — APPROVED** [review]
+1

**Comment by cstamas** [comment]
Actually, all this "discussion" is IMHO fluke:
* we discuss a change in Apache Maven sources, what it has to do with a (publicly available, thanks!) service name that a privately held company operate and publicize?
* Super POM points to https://repo.maven.apache.org/maven while https://central.sonatype.org/ is being mentioned here. What first thing has to do with second? (true, content-wise is "same thing", but is merely technical detail that former URL is backed by service operated by same company backing latter URL as well).
* the repository as known today (aka "central place for binary artifacts", etc, any def you can come up for that repo) was conceived and laid down by Apache Maven project.
* the name "maven central" name **was** registered back in 2011 (when issue MNG-5164 was created), but seems abandoned since, for over 7 years.

In other words, if Sonatype tomorrow renames "Central" to "Ukulele" for whatever reason, will that imply Super POM change in Apache Maven as well?

**Review by cstamas — APPROVED** [review]

**Comment by mosabua** [comment]
Hahah .. about search.maven.org .. oh well. 

The reason central.sonatype.org is important is that it is the reference documentation for how to publish ..  maybe Sonatype and the Maven project should agree on one name and use it consistently..

**Comment by michael-o** [comment]
@cstamas As you pointed out on central.s.o "The Central Repository" is a stupid, generic name not telling its purpose at all. Central for what? While Maven Central Repository clearly says that it hosts something for Maven.

As for the rest: while some has approved the PR, others didn't really make a clear statement. What is the concensus?

**Review by slachiewicz — APPROVED** [review]

**Comment by slachiewicz** [comment]
We have also this we have also this file published on maven website so after merge we should refresh it manually

**Comment by jira-importer** [comment]
Resolve #7673
